.bazelrc

```diff
@@ -14,10 +15,13 @@ common --remote_timeout=3600
 common --build_metadata=REPO_URL=https://github.com/player-ui/tools.git
 common --remote_cache_compression --remote_cache_async
 
+# Python config
+common --incompatible_default_to_explicit_init_py
+
 # CI Config 
-common:ci --build_metadata=ROLE=CI --workspace_status_command=./scripts/workspace-status.sh
```

Comment:
At this point, w/ all the Circle CI based Bazel repos, we should try to consolidate and maintain a bazel rc for these. Would be cool to do for general, non Circle CI repos if possible as well.